app/src/main/res/layout/view_user_item.xml

```diff
@@ -11,67 +11,81 @@
     android:paddingBottom="2dp"
     card_view:cardUseCompatPadding="true">
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="horizontal"
+        >
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/ivAvatar"
+        android:layout_width="wrap_content"
```

Comment:
To avoid blinking when images are loaded, set  `android:layout_width` and `android:layout_height` to `36dp`.

Reply:
Updated. Using the dimension from values/dimens.xml. In future, we can have separate buckets for different screen sizes for better support on other devices like tablet.